sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/BlobCopyOperation.java

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob.models;
+
+import com.azure.core.annotation.Immutable;
+import com.azure.storage.blob.BlobAsyncClient;
+import com.azure.storage.blob.BlobClient;
+
+import java.net.URL;
+import java.util.Objects;
+
+/**
+ * The status of a long running Azure Blob copy operation.
+ *
+ * @see BlobAsyncClient#beginCopyFromUrl(URL)
+ * @see BlobClient#beginCopyFromUrl(URL)
+ */
+@Immutable
+public class BlobCopyOperation {
```

Comment:
Should I update this to BlobCopyInfo to match [FileCopyInfo](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/models/FileCopyInfo.java)?